corc-cascading/pom.xml

```diff
@@ -13,8 +13,8 @@
   <description>An ORC File Scheme for Cascading</description>
```

Comment:
Change the version of this project to 3.0.0-SNAPSHOT since there are non-backwards compatible changes due to the underlying libraries.